src/glossary.rst

```diff
@@ -5848,6 +6000,26 @@ parameter]s` that describes the circumstances under which a :t:`generic type`
 is a :t:`subtype` of an instantiation of itself with different :t:`[generic
 argument]s`.
 
+.. _fls_RIe80XOF8VlA:
+
+variadic part
+^^^^^^^^^^^^^
+
+:dp:`fls_ePnTyLoqJ1i7`
+A :dt:`variadic part` indicates the presence of :t:`C`-like optional
+parameters.
+
+:dp:`fls_z9D86gBFbKB5`
+See :s:`VariadicPart`.
+
+.. _fls_AYmsHgbNcW8r:
+
+variant conformant
+^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_MfiPifTsTzgM`
+HELP LUKAS!
```

Comment:
The usage of this term in types and trait is wrong, I think this should be variance conformant.

Reply:
Removed.